tools/clang-tidy-plugin/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ include(ExternalProject)
 find_package(LLVM REQUIRED CONFIG)
 find_package(Clang REQUIRED CONFIG)
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
For local testing, when bringing in the outside CMakeLists.txt and the general cdda flags is not desirable.